README.md

```diff
@@ -51,10 +51,15 @@ var options = {
   delay: 0,
   easing: 'linear',
   offset: 0,
-  keyboardNavigation: {
-    enabled: false,
-    nextPanelKey: 40,
-    previousPanelKey: 38,
+  navigation: {
+    keys: {
+      nextKey: 40,
+       prevKey: 38,
+    },
+    buttons: {
+      $nextButton: $('#nextButton'),
+      $prevButton: $('#prevButton'),
```

Comment:
This should represent the default value. By default the navigation is disabled which means they should be false.